periodic/periodic.go

```diff
@@ -70,6 +70,31 @@ func (r *RunnerOptions) MakeRunners(rr Runnable) {
 	}
 }
 
+// Aborter is the object controlling Abort() of the runs.
+type Aborter struct {
+	StopChan   chan struct{}
+	AbortMutex sync.Mutex
```

Comment:
Change  
AbortMutex sync.Mutex
to
sync.Mutex

Using anonymous fields

Reply:
you're right ! much better that way ! thanks !